com.unity.template-universal/Packages/com.unity.template.universal/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ All notable changes to this project template will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [11.0.0] - 2020-10-21
```

Comment:
Note that I had to add this entry to fix the URP job.
I think it was forgotten by mistake when we bumped all the changelogs.
With that fix, the publish job works.